widgets/linechart/internal/axes/value.go

```diff
@@ -36,6 +38,9 @@ type Value struct {
 	// NonZeroDecimals indicates the rounding precision used, it is provided on
 	// a call to newValue.
 	NonZeroDecimals int
+	// Formatter will format value to a string representation of the value,
+	// if Formatter is not present it will fallback to default format.
+	Formatter valueFormatter
```

Comment:
Does this field need to be exported on the struct or can it be private? Will any of the call sites need to access it?